Package.swift

```diff
@@ -16,27 +16,27 @@ let package = Package(
     targets: [
         .binaryTarget(
             name: "GoogleMobileAds",
-            path: "GoogleMobileAds/8.8.0/GoogleMobileAds.xcframework"
+            path: "GoogleMobileAds/10.12.0/GoogleMobileAds.xcframework"
         ),
         .binaryTarget(
             name: "GoogleAppMeasurement",
-            path: "GoogleMobileAds/7.69.0/GoogleAppMeasurement.xcframework"
+            path: "GoogleMobileAds/10.12.0/GoogleAppMeasurement.xcframework"
         ),
         .binaryTarget(
             name: "GoogleUtilities",
-            path: "GoogleMobileAds/7.69.0/GoogleUtilities.xcframework"
+            path: "GoogleMobileAds/10.12.0/GoogleUtilities.xcframework"
         ),
         .binaryTarget(
             name: "PromisesObjC",
             path: "GoogleMobileAds/7.69.0/PromisesObjC.xcframework"
```

Comment:
Aren't `PromisesObjC` getting updated too?

Reply:
The zip that comes from Google doesn't have that library so I didn't update that one.
![Screenshot 2023-10-27 at 14 50 53](https://github.com/dowjones-mobile/external-binaries/assets/560069/d63ac303-ba0c-4bbd-bea4-2b3c257ab640)